Assets/BossRoom/Scripts/Server/Game/Entity/ServerEnemyPortal.cs

```diff
@@ -110,10 +116,35 @@ IEnumerator CoroGoDormantAndThenRestart()
                     serverComponent.Unbreak();
                 }
             }
+
             m_State.IsBroken.Value = false;
             m_WaveSpawner.SetSpawnerEnabled(true);
             m_CoroDormant = null;
         }
+
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+        public void ForceRestart()
+        {
+            if (m_CoroDormant != null)
+            {
+                StopCoroutine(m_CoroDormant);
+            }
+            Restart();
+        }
+
+        public void ForceDestroy()
+        {
+            foreach (var state in m_BreakableElements)
+            {
+                if (state)
+                {
+                    var serverComponent = state.GetComponent<ServerBreakableLogic>();
+                    Assert.IsNotNull(serverComponent);
+                    serverComponent.ReceiveHP(null, Int32.MinValue);
```

Comment:
Just a thought: already destroyed spawners won't restart their cooldown when the cheat directs all to be destroyed. This means spawners would come back to normal at different intervals. Not sure if that is even a requirement, since you could just cheat revive the spawners and cheat destroy them again to sync their cooldowns.

Again, this I don't think is a requirement for this cheat but just a consideration for this cheat's usage.

Reply:
Should this be documented with comments?

Reply:
Yes, leave a remark on the cheat code method that this is a possible outcome. I don't think it's necessary to retrofit existing code for this type of parity just for a cheat.